sdk/workflow/update.go

```diff
@@ -0,0 +1,90 @@
+//  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+//  the License. You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+//  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+//  specific language governing permissions and limitations under the License.
+
+package workflow
+
+import "github.com/conductor-sdk/conductor-go/sdk/model"
+
+
+type UpdateTask struct {
+	Task
+}
+
+//  Create a new Update with Task Id (targetTaskId)
+func NewUpdateTaskWithTaskId(taskRefName string, status model.TaskResultStatus, targetTaskId string) *UpdateTask {
+
+	update := &UpdateTask{
+			Task: Task{
+				name:              string(UPDATE),
+				taskReferenceName: taskRefName,
+				taskType:          UPDATE,
+				inputParameters:  map[string]interface{}{
+                    "taskId": targetTaskId,
+                    "taskStatus": status,
+                },
+        },
+    }
+
+	return update
+}
+
+//  Create a new Update Task with workflow Id (targetWorkflowId) and  task ref (targetTaskRefName)
+func NewUpdateTask(taskRefName string, status model.TaskResultStatus, targetWorkflowId string, targetTaskRefName string) *UpdateTask {
+
+	update := &UpdateTask{
+			Task: Task{
+				name:              string(UPDATE),
+				taskReferenceName: taskRefName,
+				taskType:          UPDATE,
+				inputParameters:  map[string]interface{}{
+                    "workflowId": targetWorkflowId,
+                    "taskRefName": targetTaskRefName,
+                    "taskStatus": status,
+                },
+			},
+        }
+
+	return update
+}
+
+func (task *UpdateTask) MergeOutput(value interface{}) *UpdateTask {
```

Comment:
Why is `value` value `interface{}`? Accepted values are `true` or `false`.

This would allow me to do `updateTask.MergeOutput("whatever")`. Is that OK?

Reply:
No, it should be bool. Good catch